src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -54,12 +54,19 @@
   (with-open [rs (.getCatalogs metadata)]
     (set (map :table_cat (jdbc/metadata-result rs)))))
 
-(defn- database-type->base-type-or-warn
+(defn database-type->base-type-or-warn
   "Given a `database-type` (e.g. `VARCHAR`) return the mapped Metabase type (e.g. `:type/Text`)."
-  [driver database-type]
+  [driver namespaced-col database-type]
   (or (sql-jdbc.sync.interface/database-type->base-type driver (keyword database-type))
-      (do (log/warnf "Don't know how to map column type '%s' to a Field base_type, falling back to :type/*."
-                     database-type)
+      (do (let [pretty-column (str/join "."
+                                        (reverse
+                                         (into []
+                                               (comp (take-while some?)
+                                                     (map #(str "'" % "'")))
```

Comment:
```suggestion
                                                     (map #(str "`" % "`")))
```

or `metabase.driver.sql.util/quote-name` ? not that proper quoting matters here, but it might be more concise.